### ISSUE TITLE
diffing file: echos annoying message

### ISSUE THREAD

**Issue by davidsu** [neoclide/coc-eslint#95]
First of all, thanks for the AMAZING plugin!

using fugitive to diff a file, left file's path is `fugitive:///$PROJ_PATH/.git//0/$FILE_PATH.ts`
echos `Failed to load the ESLint library for the document /^fugitive:/...`
the filetype is `typescript` as should be for syntax highlight.
how do I disable for `fugitive` files or supress messages for it?

**Comment by chemzqm** [comment]
It should be bug of coc-eslint.

**Comment by davidsu** [comment]
Thanks for the quick reply.
if anyone comes here with the same problem, for now I worked around by doing
```viml
function! GitDiff()
    let g:coc_disable_uncaught_error = 1
    Gdiff
endfunction
function! EndDiff()
    let g:coc_disable_uncaught_error = 0
endfunction
nmap <silent><space>gd :call GitDiff()<cr>
"ed for endDiff, hacky but it's been serving me well for years
nmap <space>ed <C-w><C-j><C-w><C-l><C-w><C-o>:call EndDiff()<cr>
```

**Comment by chemzqm** [comment]
Fixed on 1.4.0
